### DISCUSSION TITLE
Questions about function calls with VLLM and Ollama

### DISCUSSION THREAD

**Discussion by zzllkk2003** [spring-projects/spring-ai#988] [Q&A]
1. Does Spring AI currently support VLLM ? If not, when is the expected timeline for supporting it?
2.  Does Spring AI facilitate function calls from Ollama? Is this feature anticipated to be included in the M2 version?

**Comment by ThomasVitale** [comment]
About question 2, work is in progress to add function calling capability in the Ollama integration. You can follow the status here: https://github.com/spring-projects/spring-ai/issues/720
